libs/sdk-backend-tiger/src/backend/features/index.ts

```diff
@@ -0,0 +1,44 @@
+// (C) 2020-2022 GoodData Corporation
+import { IStaticFeatures, IUserProfile, ILiveFeatures } from "@gooddata/api-client-tiger";
+import { TigerAuthenticatedCallGuard } from "../../types";
+import { ITigerFeatureFlags, DefaultFeatureFlags } from "../uiFeatures";
+
+import { getFeatureHubFeatures } from "./hub";
+import { getStaticFeatures } from "./static";
+
+export class TigerFeaturesService {
+    constructor(private readonly authCall: TigerAuthenticatedCallGuard) {}
+
+    public async getFeatures(profile: IUserProfile): Promise<ITigerFeatureFlags> {
+        return this.authCall(async (client) => {
+            const prof = profile || (await client.profile.getCurrent());
```

Comment:
If `profile` can be undefined, it should be marked as such in the `getFeatures` signature.